cdk/java/README.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to your CDK Java project!
+
+This is a blank project for CDK development with Java.
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app.
+
+It is a [Maven](https://maven.apache.org/) based project, so you can open this project with any Maven compatible Java IDE to build and run tests.
+
+## Useful commands
+
+ * `mvn package`     compile and run tests
```

Comment:
Here I would only include the commands for building, testing, and deploying the stack (while the generic CDK commands are already documented [here](https://github.com/alexcasalboni/aws-lambda-power-tuning/tree/master/cdk))